save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/AvatarController.kt

```diff
@@ -73,30 +69,23 @@ internal class AvatarController(
     )
     @ApiResponse(responseCode = "200", description = "Returns content of the file.")
     @ApiResponse(responseCode = "404", description = "Execution with provided ID is not found.")
-    @GetMapping("/{organizationName}/{imageName}")
+    @GetMapping("/organizations/{organizationName}")
     fun forOrganization(
         @PathVariable organizationName: String,
-        @PathVariable imageName: String,
     ): Mono<ByteBufferFluxResponse> = AvatarKey(
         type = AvatarType.ORGANIZATION,
         objectName = organizationName,
-        imageName = imageName
     )
         .toMonoResponse()
         .switchIfEmptyToNotFound {
-            "Not found avatar for organization $organizationName with name $imageName"
+            "Not found avatar for organization $organizationName"
         }
 
     private fun AvatarKey.toMonoResponse(): Mono<ByteBufferFluxResponse> = this.toMono()
         .filterWhen(avatarStorage::doesExist)
-        .flatMap {
-            avatarStorage.lastModified(this)
-                .map { lastModified ->
-                    ResponseEntity.ok()
-                        .cacheControl(CacheControl.maxAge(longExpirationTime.toJavaDuration()).cachePublic())
```

Comment:
why do we removed caching?

Reply:
The cache was looking at changing the file path. But now the path does not change because the file name is equal to the organization. Only the content changes. Therefore, when changing the avatar, it did not change on the front.

Reply:
so looks like we missed caching functionality for avatar now. Is there way to re-implement it with same file-name?
maybe introduce a version of file and use last modified time as version?

Reply:
looks like request with `cache-control` supports reload: https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#reload_and_force_reload

Reply:
Added version tag to url. In the next PR I will delete ImageInfo and change the cache.